packages/aws-cdk/lib/init-templates/v1/app/javascript/cdk.template.json

```diff
@@ -1,3 +1,13 @@
 {
-  "app": "node bin/%name%.js"
+  "app": "node bin/%name%.js",
+  "watch": {
+    "include": ["**"],
+    "exclude": [
+      "jest.config.js",
+      "package*.json",
```

Comment:
Let's maybe add `yarn.lock` here too?

Same comment in the other JS/TS templates.